Prox/Prox/Map View/MapViewController.swift

```diff
@@ -3,14 +3,65 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
+import GoogleMaps
 
 class MapViewController: UIViewController {
 
-    init() {
-        super.init(nibName: nil, bundle: nil)
-    }
+    lazy var rootContainer: UIStackView = {
```

Comment:
The `lazy var` convention appears to be done so you can declare the variables/methods in the file in any order (whereas you have to wait for references to be defined first if it's non-lazy).